client/jetpack-cloud/sections/partner-portal/credit-card-fields/payment-method-image.tsx

```diff
@@ -1,11 +1,10 @@
 export default function PaymentMethodImage() {
 	return (
 		<svg
-			width="432"
-			height="296"
 			viewBox="0 0 432 296"
 			fill="none"
 			xmlns="http://www.w3.org/2000/svg"
+			preserveAspectRatio="xMidYMid meet"
```

Comment:
This is not directly related to this PR, since it didn't introduce `payment-method-image.tsx`, but the SVG uses `feFlood` [which has "unknown support" in Safari](https://caniuse.com/?search=feflood). So in terms of browser support, then this might have some issues.

I cannot see it on my own (latest) version of Safari, but worth pointing out.